<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1579</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>新增</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditList">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddRow">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>新增一行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelRow">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除指定行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddCol">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>新增一列</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelCol">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除指定列</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTable">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>460</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>新增列名字</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>389</width>
      <height>179</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="labelChoice">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>249</width>
      <height>129</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择角色：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>249</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddTab">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>AddTab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelTab">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>DelTab</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1579</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
